src/main/flight/imu.h

```diff
@@ -28,6 +28,11 @@ extern int accSumCount;
 extern float accVelScale;
 extern int32_t accSum[XYZ_AXIS_COUNT];
 
+typedef struct {
+    float w,x,y,z;
+    float ww,wx,wy,wz,xx,xy,xz,yy,yz,zz;
+} quaternion;
+#define DEFINE_QUATERNION(X) STATIC_UNIT_TESTED quaternion X = {.w=1, .x=0, .y=0,.z=0, .ww=1, .wx=0, .wy=0, .wz=0, .xx=0, .xy=0, .xz=0, .yy=0, .yz=0, .zz=0}
```

Comment:
More C-like ... then working C code (How that ?)
Defining a quaternion inside a function  would be  a question of visibility ...
unittest uses only a extern reference to the main quaternion "q" .